ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -130,5 +130,10 @@ def __sub__(self, y, name="sub"):
     def __truediv__(self, y, name="truediv"):
         return y.__rtruediv__(self.data)
 
+    def __len__(self):
+        raise ivy.exceptions.IvyError(
+            "len is not well defined for a symbolic Tensor. Please call `x.shape` rather than `len(x)` for shape information."
+        )
```

Comment:
This causes a lint error "E501 line too long (126 > 88 characters)". 
There is a pre-commit hook you can use to avoid having to worry about this type of errors. You can read about it here https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit.
In this case of course you can just fix it manually 'cause it's just one line 🙂